tests/suites/test_suite_x509write.function

```diff
@@ -117,13 +119,23 @@ void x509_csr_check( char * key_file, char * cert_req_check_file, int md_type,
         TEST_ASSERT( buf[buf_index] == 0 );
     }
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    // When using PSA crypto, RNG isn't controllable, so cert_req_check_file can't be used
+    (void)f;
+    (void)olen;
+    (void)check_buf;
```

Comment:
Alternatively, we could just wrap the corresponding declarations with an `#if` - except of course for `cert_req_check_file` which is a parameter.